.travis.yml

```
@@ -1,10 +1,20 @@
 language: julia
+
 os:
+  - osx
   - linux
+  - windows
+  - freebsd
```

```suggestion
  - linux
```
I'm fine with just Linux, this package does no platform-specific things, so testing on Linux is sufficient and a lot faster too.